content/sensu-go/5.19/operations/deploy-sensu/install-sensu.md

```diff
@@ -26,19 +26,31 @@ See [supported platforms][5] for more information.
 
 ## Architecture overview
 
+Sensu works differently from other monitoring and observability solutions.
+
+The **Sensu backend** is powered by an an embedded transport and [etcd][16] datastore.
+You do not need to provision each device, server, container, or sidecar you want to monitor or list each of them in the Sensu backend.
+Instead, you create [subscriptions][41] and define which checks to run for all infrastructure components that belong to each subscription.
```

Comment:
Subscriptions aren’t a real resource in Sensu. A User creates checks that target one or more subscriptions. Agents subscribe to subscriptions that align with their host’s responsibilities/roles.